### ISSUE TITLE
Wrong lens path for puppet enterprise

### ISSUE THREAD

**Issue by mwhahaha** [camptocamp/puppet-augeas#52]
A user reported an issue with the lens location on puppet enterprise, https://github.com/mwhahaha/puppet-pgpool/issues/12. It looks like https://github.com/camptocamp/puppet-augeas/blob/master/manifests/params.pp#L6 might need a conditional for puppet enterprise as the path might need to be /opt/puppet/share/augeas/lenses/dist/.

**Comment by isaiahfrantz** [comment]
This still affects puppet opensource:
puppet-agent.x86_64                  1.2.5-1.el6 
ruby-augeas.x86_64                   0.4.1-3.el6

**Comment by isaiahfrantz** [comment]
I upgraded puppet-agent to 1.2.7 and still see this issue:
puppet-agent.x86_64                  1.2.7-1.el6
ruby-augeas.x86_64                   0.4.1-3.el6

It continues to look in /var/lib/puppetlabs/lib/augeas/lenses for lenses (according to debug mode) but the lenses are actually installed under /opt/puppetlabs/puppet/share/augeas/lenses/dist/. 
I tried to rm the default and symlink to the real dir but all my lenses got deleted and the default files repopulated:
-rw-r--r--. 1 root root 19697 Mar 11 17:03 functions
-rw-r--r--. 1 root root   387 Mar 11 17:03 init_d_functions.aug
drwxr-xr-x. 2 root root  4096 Mar 11 17:03 tests

Ive looked around but cant find any info on configuring augeas where to look for lenses.
Any thought?
